frontend/ui/widget/titlebar.lua

```diff
@@ -82,6 +83,7 @@ function TitleBar:init()
         self.width = Screen:getWidth()
     end
     local title_max_width = self.width - 2 * self.title_h_padding
+    local subtitle_max_width = title_max_width
```

Comment:
This might overflow on the right when has_left_icon (as I add some horizontal span on the left of the whole title_group thing). So, if it's fine having subtitle text below the right icon, you can substract `- left_icon_size - self.button_padding` in the `if self.left_icon` then.

Otherwise, to allow subtitle to go below both icons, I'd need to rethink their layouts.
I thought about it when redoing ImageViewer, previously we had:
![image](https://user-images.githubusercontent.com/24273478/148694422-4cf4e352-910b-4760-8de9-80eb702674f6.png)
Now we will have:
![image](https://user-images.githubusercontent.com/24273478/148694876-d7321173-64b9-4ec0-8ae7-c508b6f77293.png)

It bothered me initially to get this indentation, but I ended up liking it :) I'm also using titlebar_width, so my subtitle wraps before the right button, should it go further on the right below the right icon, or stop before if we have also a left button, to keep things balanced ? What do you think?
(I'll PR my stuff after this PR is merged, as we would be conflicting as I'm adding a `subtitle_multilines`. And I've done KeyValuePage.)

Reply:
The path in the subtitle is a special case, and it must be consistent with file manager home screen, I think.

Reply:
> This might overflow on the right when has_left_icon

Yeah, I'm going to add Home button to the path chooser.

Reply:
> Or we can make the path shorter too. Can we calculate the width for file manager?

Which "width" do you mean ? If making it shorter (so, fitting in between the 2 icons), that would be using title_max_width as previously.
Or may be there's a bug: I compute title_max_width by remoing icons width, but in the end, I have it centered, so it may overflow below the right button if no left button.

> The path in the subtitle is a special case

That would need rethinking the widget layout for self.title_group, we could have another option to build things differently (subtitle using the full width, or being constrained between the 2 buttons - title always is). I can think about it and try and add that in my next PR.

Reply:
> Which "width" do you mean ?

I mean the width of the path in FM (where the icons are bigger) to be of the same width as the path in the PathChooser,

Reply:
Isn't it the full screen width (minus some padding) in File manager?
![image](https://user-images.githubusercontent.com/24273478/148697966-6f3848c6-128d-4050-9c74-fc8dd06aee49.png)

I think we would get this (width minus 2*title_h_padding) in TitleBar when no button.

Reply:
So, new options `subtitle_multilines` and `subtitle_fullwidth`.

Reply:
Implemented in #8640.
I picked your subtitle_truncate_left, so you can reset titlebar.lua in this PR.